apps/judicial-system/backend/migrations/20241211095654-update-defendant.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.sequelize.transaction((t) =>
+      Promise.all([
```

Comment:
You don't technically need a Promise.all here because you're doing just one thing, so you could leave that out and just have queryInterface.addColumn(... straight away.